src/plays/typing-speed-test/components/Word.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Word = ({ text, isActive, correct }) => {
+  // if word is correct
+  if (correct === true)
+    return <span className="text-green-500 text-[1.2rem]">{text} </span>;
+
+  // if word is incorrect
+  if (correct === false)

```

Comment:
```suggestion
  if (!correct)

```

Reply:
hey @Sachin-chaurasiya here i have to exactly check for `correct === false` because for the first word it will get `undefined` because user has not typed anything yet so `typedWordsArray` will be empty and for every word `Word` component will return text with red color.

Have a look
![image](https://user-images.githubusercontent.com/28287599/192333367-346fcce1-62e9-4675-882c-de75d173bd8c.png)

Reply:
@makdoom , `correct` will have the boolean value either `true` or `false` right? so we can just use the negation there,  can make the change and try that?

Reply:
Done

Reply:
hey @Sachin-chaurasiya I've tried that when I changed `correct === false` to `!correct` I am getting all the words in red colour. because on initial render the `correct` prop will get `undefined` so it is rendering the `(!correct)` condition.

![image](https://user-images.githubusercontent.com/28287599/193452845-1d3176e3-81e2-48d2-bb7e-bda05a409eb2.png)

 `code:`
![image](https://user-images.githubusercontent.com/28287599/193452879-1cb36d05-554c-44b6-a769-0fc11f741e48.png)